### PR TITLE
fix(@angular-devkit/build-optimizer): fix relative pure imports

### PR DESCRIPTION
**Pull request by filipesilva**

### CODE REVIEW

File: packages/angular_devkit/build_optimizer/src/purify/purify_spec.ts [hunk 1]
```diff
@@ -27,17 +27,36 @@ describe('purify', () => {
 
   it('prefixes safe default imports with /*@__PURE__*/', () => {
     const input = tags.stripIndent`
-      /* harmony import */ var __WEBPACK_IMPORTED_MODULE_4_rxjs_Subject__ = __webpack_require__("rlar");
-      /* harmony import */ var __WEBPACK_IMPORTED_MODULE_4_rxjs_Subject___default = __webpack_require__.n(__WEBPACK_IMPORTED_MODULE_4_rxjs_Subject__);
+    /* harmony import */ var __WEBPACK_IMPORTED_MODULE_4_rxjs_Subject__ = __webpack_require__("rlar");
```
Comment (IgorMinar) [inline packages/angular_devkit/build_optimizer/src/purify/purify_spec.ts]: this seems like an incorrect indentation change

File: packages/angular_devkit/build_optimizer/src/purify/purify_spec.ts [hunk 2]
```diff
@@ -27,17 +27,36 @@ describe('purify', () => {
...
+        /* harmony import */ var __WEBPACK_IMPORTED_MODULE_2__symbol_observable__ = __webpack_require__(45);
+        /* harmony import */ var __WEBPACK_IMPORTED_MODULE_3__util_pipe__ = __webpack_require__(71);
+        /** PURE_IMPORTS_START ._util_root,._util_toSubscriber,.._symbol_observable,._util_pipe PURE_IMPORTS_END */
+      `;
+    const output = tags.stripIndent`
+        /* harmony import */ var __WEBPACK_IMPORTED_MODULE_0__util_root__ = /*@__PURE__*/__webpack_require__(13);
+        /* harmony import */ var __WEBPACK_IMPORTED_MODULE_1__util_toSubscriber__ = /*@__PURE__*/__webpack_require__(67);
+        /* harmony import */ var __WEBPACK_IMPORTED_MODULE_2__symbol_observable__ = /*@__PURE__*/__webpack_require__(45);
+        /* harmony import */ var __WEBPACK_IMPORTED_MODULE_3__util_pipe__ = /*@__PURE__*/__webpack_require__(71);
+        /** PURE_IMPORTS_START ._util_root,._util_toSubscriber,.._symbol_observable,._util_pipe PURE_IMPORTS_END */
```
Comment (IgorMinar) [inline packages/angular_devkit/build_optimizer/src/purify/purify_spec.ts]: where do the dots come from? can you leave a comment in the code explaining that?

File: packages/angular_devkit/build_optimizer/src/purify/purify_spec.ts (hunk 1 above)
Comment (filipesilva) [reply packages/angular_devkit/build_optimizer/src/purify/purify_spec.ts]: Fixed

File: packages/angular_devkit/build_optimizer/src/purify/purify_spec.ts (hunk 2 above)
Comment (filipesilva) [reply packages/angular_devkit/build_optimizer/src/purify/purify_spec.ts]: Fixed

File: packages/angular_devkit/build_optimizer/src/transforms/prefix-functions.ts [hunk 3]
```diff
@@ -101,9 +101,12 @@ export function findPureImports(parentNode: ts.Node): string[] {
     if (node.kind === ts.SyntaxKind.ImportDeclaration
       && (node as ts.ImportDeclaration).importClause) {
 
-      // Save the path of the import transformed into snake case
+      // Save the path of the import transformed into snake case and remove relative paths.
```
Comment (hansl) [inline packages/angular_devkit/build_optimizer/src/transforms/prefix-functions.ts]: So:

```
import {} from '@someModule/my-component';
```

and 

```
import {} from './someModule/my-component';
```

would have the same id?

**Pull request merged**